### PR TITLE
Spellcheck: remove modals

### PR DESCRIPTION
**Pull request by thenor**
"Don't", "shouldn't", "didn't" etc. are caught in spellchecker and should be
avoided in the documentation.

Signed-off-by: Therese Nordqvist <therese.nordqvist@pelagicore.com>

### CODE REVIEW

**Review by sashko — COMMENTED** [review]
I've seen this yesterday. We should figure out why apostrophe isn't recognized instead.

File: docs/articles/sde/introduction_to_sde.rst [hunk 1]
```diff
@@ -18,7 +18,7 @@ Dependencies:
 * VirtualBox
 * Self-extracting SDK package to install into the SDE. See, :ref:`building-the-sdk`.
 
-.. note:: Ubuntu 16.04 default repositories doesn't provide the latest version
+.. note:: Ubuntu 16.04 default repositories does not provide the latest version
```
Comment (TSAnsari) [inline docs/articles/sde/introduction_to_sde.rst]: should be "do not"

**Comment by sashko** [comment]
@thenor push the Jenkinsfile change and let's try to figure out what's wrong with the apostrophe on Monday.

File: docs/articles/sde/introduction_to_sde.rst (hunk 1 above)
Comment (thenor) [reply docs/articles/sde/introduction_to_sde.rst]: Yes. Missed this one. I will fix.

**Comment by thenor** [comment]
I will close this and we can revisit the change later. I think it should be done, not because the spellchecker fails, but because modals shouldn't be used in documentation. On the other hand if we want to investigate why it fails then we can do that first.

**Comment by martin-ejdestig** [comment]
> I've seen this yesterday. We should figure out why apostrophe isn't recognized instead.

I would argue the opposite. We should commit this change regardless of what we do with the spell checker. I do not think it is appropriate to use contractions in formal/technical writing.

>  I think it should be done, not because the spellchecker fails, but because modals shouldn't be used in documentation.

Yes. +1 for this.

**Review by sashko — APPROVED** [review]

**Pull request merged**